commit 5c6c59dc12b9870b836c0e30854df990db4d443e
Author: Aleksander Machniak <machniak@kolabsys.com>
Date:   Thu Oct 25 08:30:37 2018 +0000

    Fix selected folder color in Files/Calendar/Tasks (T4371)
    
    + some other small fixes

diff --git a/skins/chameleon/styles.less b/skins/chameleon/styles.less
index e18c018..7ba1f44 100644
--- a/skins/chameleon/styles.less
+++ b/skins/chameleon/styles.less
@@ -174,6 +174,7 @@ body.login #header {
 #mainscreencontent,
 #tasksview,
 .tasklistview #tagsbox,
+#tasklistbox,
 #folderlistbox,
 #filelistcontainer,
 #fileinfobox,
@@ -435,6 +436,10 @@ body.login #header {
 	background-position: -5px -2959px;
 }
 
+#ktaskpopup a.button-chat .button-inner {
+	background-image: none; /* TODO */
+}
+
 .ktaskmenu .dropdownhandle {
 	position: absolute;
 	display: block;
@@ -614,7 +619,8 @@ ul.listing.focus .listitem.focused span {
 	outline: none;
 }
 
-ul.treelist li.selected a:focus {
+ul.treelist li.selected > div > a:focus,
+ul.treelist li.selected > a:focus {
 	color: @list-selected-text !important;
 }
 
@@ -2227,14 +2233,34 @@ a.button.changeformat.text.selected span.icon {
 	color: @text-color;
 }
 
-#tasklistsbox .treelist li.selected > div span.listname,
-#calendars .treelist li.selected > div span.calname,
-#directorylist li.addressbook.selected > div a,
-.notesview #notebooks li.selected > div .listname,
-#files-folder-list ul li.selected > span.name {
+.treelist li > div > a {
+	color: @list-text !important;
+}
+
+.treelist li > div.virtual > a {
+	color: #aaa !important;
+}
+
+.treelist li > div > a,
+#files-folder-list ul li > .name {
+	top: 3px !important;
+}
+
+.treelist li.selected > div > a,
+#files-folder-list ul li.selected > .name {
 	color: @list-selected-text !important;
 }
 
+.treelist li.selected > div > a,
+#files-folder-list ul li.selected > .name {
+	color: @list-selected-text !important;
+}
+
+html.mozilla .treelist li > div > input {
+	top: 2px !important;
+	left: 15px !important;
+}
+
 #directorylist li.addressbook.readonly {
 	background-position: 98% -2121px;
 }
